README.md

```diff
@@ -4,6 +4,8 @@
 
 Offical [Apache CouchDB](https://couchdb.apache.org/) library for [Node.js](https://nodejs.org/).
 
+> Note: Nano >=11.0.0 is for Node 18/20 and above. If you are using Node 16 or older, you need Nano 10.1.2.
```

Comment:
I feel we should be a bit more clear that 11+ is a breaking changes release because of the http lib update

Reply:
I've beefed up the messaging at the top.